integration/js/utils/SdkBaseTest.js

```diff
@@ -15,6 +15,9 @@ class SdkBaseTest extends KiteBaseTest {
     if (testName === 'ContentShareOnlyAllowTwoTest') {
       this.url = this.getTransformedURL(this.url, 'max-content-share', 'true');
     }
+    if (testName.includes('ContentShare')) {
+      this.capabilities['extendedDebugging'] = false;
+    }
```

Comment:
I don't think this worked as stated previously. This is to set the capabilities in the web driver where as extend-debugging is SauceLabs option